lib/update_remote_id.rb

```diff
@@ -0,0 +1,51 @@
+# Update events remote_id from existing schedule xml data
```

Comment:
I'll delete this after I've updated the live database. The modified importer will set the remote id if schedule.xml data is available.